.github/workflows/main.yml

```diff
@@ -0,0 +1,107 @@
+# This GitHub Actions workflow contains four jobs:
+# -- First, Lint Code uses the latest Ubuntu image and Python 3.9 to check
+# the code for defects, using the Poetry lint task.
+# -- Second, Lint Markdown uses the latest Ubuntu image and makrdownlint to
+# check markdown files for defects.
+# -- Third uses the latest Ubuntu image and cspell to check for spelling
+# defects in the markdown.
+# -- Fourth, Test uses a strategy matrix to test the code with different
```

Comment:
`Test` also uses different operating systems, which may be a good detail to mention here.